package.json

```diff
@@ -51,11 +52,15 @@
     "build-webview": "npm run build --prefix=packages/salesforcedx-webview-ui",
     "test-webview": "npm run test --prefix=packages/salesforcedx-webview-ui",
     "bundle-webview":
-      "npm run compile --prefix=packages/salesforcedx-webview-ui"
+      "npm run compile --prefix=packages/salesforcedx-webview-ui",
+    "aggregateJUnit": "node scripts/aggregate-junit-xml.js"
   },
   "repository": {
     "type": "git",
     "url": "git+https://github.com/forcedotcom/salesforcedx-vscode.git"
   },
-  "license": "BSD-3-Clause"
+  "license": "BSD-3-Clause",
+  "dependencies": {
+    "junit-xml-parser": "0.0.3"
+  }
```

Comment:
I will remove this on my next commit. I tried playing around with a parser to remove querying the test dashboard for failures.